diff --git a/dist/index.js b/dist/index.js
index 187cd54ec26e275073271e8dd2311853af9a725e..fd8573fca8504abca34e1684400911bd715e3226 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -15,7 +15,8 @@ var tsExtensions = "ts,mts,cts";
 var tsxExtensions = "tsx,mtsx";
 var allJsExtensions = `${jsExtensions},${tsExtensions}`;
 var allJsxExtensions = `${jsxExtensions},${tsxExtensions}`;
-var supportedFileTypes = `**/*{${allJsExtensions},${allJsxExtensions},astro}`;
+var allSupportedExtensions = `${allJsExtensions},${allJsxExtensions},astro,svelte`;
+var supportedFileTypes = `**/*.{${allSupportedExtensions}}`;
 var testsFilePatterns = [
   `**/*.{test,spec}.{${allJsExtensions}}`,
   `**/tests/**/*.{${allJsExtensions}}`,
@@ -87,7 +88,6 @@ import tsdoc from "eslint-plugin-tsdoc";
 import unicorn from "eslint-plugin-unicorn";
 import globals from "globals";
 import tseslint3 from "typescript-eslint";
-import { fixupPluginRules } from "@eslint/compat";
 import eslintJs from "@eslint/js";
 import preferEarlyReturn from "@regru/eslint-plugin-prefer-early-return";
 import stylistic from "@stylistic/eslint-plugin";
@@ -691,10 +691,10 @@ var getBaseConfig = (userConfigChoices) => {
     },
     {
       files: [
-        "**/*.stories.@(ts|tsx|js|jsx|mjs|cjs)",
-        "**/*.story.@(ts|tsx|js|jsx|mjs|cjs)"
+        "**/*.stories.@(ts|tsx|js|jsx|mjs|cjs|svelte)",
+        "**/*.story.@(ts|tsx|js|jsx|mjs|cjs|svelte)"
       ],
-      plugins: { storybook: fixupPluginRules(storybook) },
+      plugins: { storybook },
       rules: {
         ...storybook.configs["flat/recommended"][1].rules,
         ...storybook.configs["flat/csf"][1].rules,
@@ -703,6 +703,7 @@ var getBaseConfig = (userConfigChoices) => {
     },
     {
       files: ["**/.storybook/main.@(js|cjs|mjs|ts)"],
+      plugins: { storybook },
       rules: { ...storybook.configs["flat/recommended"][2].rules }
     },
     {
@@ -721,7 +722,7 @@ var getBaseConfig = (userConfigChoices) => {
       rules: { "fsecond/prefer-destructured-optionals": 2 }
     },
     {
-      files: [`**/*.config.{${allJsExtensions}}`],
+      files: [`**/*.config.{${allJsExtensions}}`, `.storybook/*.{${allJsExtensions}}`],
       rules: {
         "import/no-default-export": 0,
         "import/no-anonymous-default-export": 0,
