diff --git a/dist/index.js b/dist/index.js
index 2b83747e0517c5883b6cadc86a6bf3b78d5abfae..30e8597ec56cebb7a2dd7926d9cb18e248fec1b0 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -16,7 +16,8 @@ var tsExtensions = "ts,mts,cts";
 var tsxExtensions = "tsx,mtsx";
 var allJsExtensions = `${jsExtensions},${tsExtensions}`;
 var allJsxExtensions = `${jsxExtensions},${tsxExtensions}`;
-var supportedFileTypes = `**/*{${allJsExtensions},${allJsxExtensions},astro}`;
+var allSupportedExtensions = `${allJsExtensions},${allJsxExtensions},astro,svelte`;
+var supportedFileTypes = `**/*.{${allSupportedExtensions}}`;
 var testsFilePatterns = [
   `**/*.{test,spec}.{${allJsExtensions}}`,
   `**/tests/**/*.{${allJsExtensions}}`,
@@ -1192,7 +1193,6 @@ var getExportableConfig = (userConfigChoices = sheriffStartingOptions, areAllRul
 };
 
 // src/index.ts
-import {} from "@typescript-eslint/utils";
 import { default as default2 } from "typescript-eslint";
 var exportableAllJsExtensions = allJsExtensions;
 var exportableAllJsxExtensions = allJsxExtensions;
