diff --git a/dist/index.js b/dist/index.js
index 2b83747e0517c5883b6cadc86a6bf3b78d5abfae..b8865ef3baa146e9eef1548510e15bf76ad70f66 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -16,7 +16,8 @@ var tsExtensions = "ts,mts,cts";
 var tsxExtensions = "tsx,mtsx";
 var allJsExtensions = `${jsExtensions},${tsExtensions}`;
 var allJsxExtensions = `${jsxExtensions},${tsxExtensions}`;
-var supportedFileTypes = `**/*{${allJsExtensions},${allJsxExtensions},astro}`;
+var allSupportedExtensions = `${allJsExtensions},${allJsxExtensions},astro,svelte`;
+var supportedFileTypes = `**/*.{${allSupportedExtensions}}`;
 var testsFilePatterns = [
   `**/*.{test,spec}.{${allJsExtensions}}`,
   `**/tests/**/*.{${allJsExtensions}}`,
